cypress/e2e/auth.cy.ts

```diff
@@ -1,11 +1,35 @@
 describe("Authentication tests", () => {
+    const email: string = Cypress.env("TEST_USER");
```

Comment:
is `: string` really needed? can this just be automatically inferred (potentially with a `!` in case of null)?

Reply:
It's inferred as any without it (which is fine, it's just nice to have everything be typed)